### ISSUE TITLE
Automatically cancelling non-pinned domains

### ISSUE THREAD

**Issue by hollanderbart** [datatheorem/TrustKit#202]
For a example project I've used the following configuration
```
            kTSKPinnedDomains: [
                "duckduckgo.com": [
                    kTSKIncludeSubdomains: true,
                    kTSKPublicKeyHashes: [
                        "5jHPoWiSLX2qeLpar+jvvxEZAkUfLNkh7yIt9ikFmB0=",
                        "DjXN9zuffa46SPrrUz+L6RAC6T2GgQQVsK+djQJMEuw="
                    ]
                ],
                "google.com": [
                    kTSKIncludeSubdomains: true,
                    
                    // Invalid pins to demonstrate a pinning failure
                    kTSKPublicKeyHashes: [
                        "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=",
                        "BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB="
                    ],
                ],
            ]]
```

Using this configuration I can perform a URLSession with this configuration to perform a request to `www.duckduckgo.com` and `www.google.com`. The first request to `www.duckduckgo.com` will work but the last one `www.google.com` will not work.

I was surprised that using the exact configuration, a request to `www.yahoo.com` will still work! The log states the following: `TrustKit log: Domain www.yahoo.com is not pinned`.

TL;DR: Is there a way to specify that all requests will default be blocked, if they're not pinned?

Thanks for your answer!

**Comment by nabla-c0d3** [comment]
Blocking requests to domains that haven't been "whitelisted" is outside of the scope of TrustKit (which only does pinning).

**Comment by vegerot12** [comment]
so how do we block communication to other domains . can you please suggest a workaround something ?
